src/components/Dropdown.js

```diff
@@ -166,12 +266,25 @@ Dropdown.propTypes = {
   header: PropTypes.string,
   className: PropTypes.string,
   showItemStatus: PropTypes.bool,
+  close: PropTypes.func,
+  closeOnItemClick: PropTypes.bool,
+  position: PropTypes.arrayOf(
```

Comment:
I think it needs to be changed now, according to new `positionX` and `positionY`